lib/collections/transform/cart.js

```diff
@@ -78,5 +129,43 @@ export const cartTransform = {
     const discountTotal = Math.max(0, subTotal - discount);
     const total = discountTotal + shipping + taxes;
     return accounting.toFixed(total, 2);
+  },
+
+  /**
+   * Aggregates the cart total by shopId
+   * @method cartTotalByShop
+   * @return {object} An object with a key for each shopId in the cart where the value is the total for that shop
+   */
+  cartTotalByShop() {
+    const subtotals = this.cartSubTotalByShop();
+    const taxes = this.cartTaxesByShop();
+    const shipping = parseFloat(this.cartShipping());
+    // no discounts right now because that will need to support multi-shop
+
+    return Object.keys(subtotals).reduce((shopTotals, shopId) => {
+      if (!shopTotals[shopId]) {
+        const shopSubtotal = parseFloat(subtotals[shopId]);
+        const shopTaxes = parseFloat(taxes[shopId]);
+        const shopTotal = shopSubtotal + shopTaxes + shipping;
+        shopTotals[shopId] = accounting.toFixed(shopTotal, 2);
+      }
+      return shopTotals;
+    }, {});
+  },
+
+  /**
+   * cart items organized by shopId
+   * @method cartItemsByShop
+   * @return {Object} Dict of shopIds with an array of items from that shop that are present in the cart
+   */
+  cartItemsByShop() {
```

Comment:
Lol, not better. Just returns an array of shop objects rather than just the items
https://github.com/reactioncommerce/reaction/blob/dc868437603183153c4f473d8978f322a2c6ef86/lib/collections/transform/order.js#L97
We can probably return stuff like tax and subtotals there too.